Cod3rsGrowth.Servico/Servicos/ServicoEmpresa.cs

```diff
@@ -19,14 +20,30 @@ public ServicoEmpresa(IValidator<Empresa> empresavalidacao, IRepositorioEmpresa
 
         public void Adicionar(Empresa empresa)
         {
-            _empresaValidacao.ValidateAndThrow(empresa);
+            var resultado = _empresaValidacao.Validate(empresa, options => options.IncludeRuleSets("Adicionar", "default"));
```

Comment:
usa validate and throw, não tem necessidade de um ruleset caso vc só chame em casos similares